extensions/src/test/resources/ActionInvokerListenerTestResources/environment.json

```diff
@@ -0,0 +1,8 @@
+{
+  "environmentVariables": [
```

Comment:
This **environmentVariables** object is used to pass the work item parameters from the main pentaho server to Chronos, and in my opinion, shouldn't be used here, as it's more complex for reading and not so straightforward to rebuild from the env. variables available inside the docker image.

For example, a much more simple approach would be: 

`{
  "actionId":"prpt.backgroundExecution",
  "actionClass":"org.pentaho.reporting.platform.plugin.SimpleReportingAction",
  "actionUser":"admin",
  "actionParams":"{ \"serializedParams\": [\"java.util.HashMap\", { \"actionUser\": \"admin\", \"user_locale\": [\"java.util.Locale\", \"en_US\"],\"outputFilePattern\": \"/home/admin/Buyer Report (sparkline report).*\", \"line\": \"Trains\", \"renderMode\": \"PARAMETER\", \"accepted-page\": 0, \"inputFile\": \"/public/Steel Wheels/Buyer Report (sparkline report).prpt\", \"uniqueFileName\": true, \"output-target\": \"table/html;page-mode=page\", \"maximum-query-limit\": 0, \"htmlProportionalWidth\": \"false\", \"query-limit-ui-enabled\": \"true\", \"lineage-id\": \"1bb60f2b-1c0d-43ac-ba5d-4003b6c7e785\", \"actionId\": \"prpt.backgroundExecution\", \"showParameters\": \"true\", \"query-limit\": 0}] \"paramsToRecreate\": [\"::session\", \"streamProvider\"]}"`